etc/profile-m-z/mpv.profile

```diff
@@ -55,6 +56,7 @@ whitelist ${HOME}/.config/mpv
 whitelist ${HOME}/.config/youtube-dl
 whitelist ${HOME}/.config/yt-dlp
 whitelist ${HOME}/.config/yt-dlp.conf
+whitelist ${HOME}/.local/state/mpv/watch_later
```

Comment:
If we're going for the ~/.local/share/state/mpv path when (no)blacklisting, this needs to be adjusted accordingly.

Reply:
adjusted to `whitelist ${HOME}/.local/state/mpv`